code/__DEFINES/_lists.dm

```diff
@@ -1,7 +1,7 @@
 /*
  * Holds procs to help with list operations
  */
-
+#define LAZYISIN(L, I) (L ? (I in L) : FALSE)
```

Comment:
why not use LAZYACCESS()? the sole difference is that it returns null incase of failure, and not FALSE which both evaluate as False in checks

Reply:
Cuz I had no idea what I was doing, and forgot about it. Now switched.